channels/stable

```diff
@@ -103,137 +103,137 @@ spec:
     networking:
       kubenet: {}
   kubernetesVersions:
-  - range: ">=1.28.0"
-    recommendedVersion: 1.28.5
```

Comment:
@moshevayner Could revert the indentation change?

Reply:
Ooof, VSCode on a new computer 🤦🏼‍♂️ 
Sure, let me take care of that.